src/ui/public/doc_table/doc_table.html

```diff
@@ -104,6 +104,11 @@
   <kbn-infinite-scroll ng-if="infiniteScroll" more="addRows"></kbn-infinite-scroll>
 </div>
 
+<div ng-if="!columns.length" class="table-vis-error">
+  <h2><i class="fa fa-meh-o"></i></h2>
```

Comment:
I naively copied the markup for the "No results found" message, do you think it's worthwhile to make the new message use the better markup suggested by @cjcenizal in this PR, and deviate from the message directly below?